Microsoft.DotNet.Interactive/Extensions/CommandLineBuilderExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CommandLine;
+using System.CommandLine.Builder;
+
+namespace Microsoft.DotNet.Interactive.Extensions
+{
+    public static class CommandLineBuilderExtensions
+    {
+        public static CommandLineBuilder ResponseFile(
```

Comment:
This method already exists `System.CommandLine`:

https://github.com/dotnet/command-line-api/blob/0fd4b6381f62993c16414b00fcb4804302d04890/src/System.CommandLine/Builder/CommandLineBuilderExtensions.cs#L100-L106

Reply:
Jon … it's a property.  The helper method sets the property.

Reply:
The method in `System.CommandLine` is also setting the property and also returning the `CommandLineBuilder` instance. The implementation is identical (other than the null check).